docs/developer-guide.md

```diff
@@ -2,8 +2,12 @@
 
 KubeKey defines task pipelines for cluster operations such as installation, scaling, uninstallation, etc. And uses SSH and Kubernetes Api to perform corresponding tasks on hosts and cluster with host grouping and configuration management.
 
-![Image](img/KubeKey-Architecture.jpg?raw=true)
+![Image](img/KubeKey-Architecture.png?raw=true)
 
+* `Action`: The most basic unit. It represents one operation performed on one node;
+* `Task`: The one that manages `Action`. It contains fields such as `Action`, nodes, whether to retry, whether to execute in parallel, etc. In short, it represents executing an `Action` on the specified nodes as required;
```

Comment:
> It contains fields such as `Action`, nodes, whether to retry, whether to execute in parallel, etc.

This sentence looks strange to me, especially the part after `Action`.

Reply:
`Task` is the structure defined in the code, the following content describes some fields contained in this structure.